cadastre/forms/cadastre_search_form.ui

```diff
@@ -398,6 +394,28 @@
           </layout>
          </widget>
         </item>
+        <item>
+         <widget class="QGroupBox" name="grpOptions">
+          <property name="title">
+           <string>Options</string>
+          </property>
+          <layout class="QVBoxLayout" name="verticalLayout_3">
+           <item>
+            <widget class="QCheckBox" name="cbForThirdParty">
```

Comment:
For new buttons, it would be nice to follow python convention for naming variable.
This is raising warning.

I will push to your branch.